front/src/components/Navbar/TranslationToggle.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+import { setLanguage } from '@/app/actions';
+import { usePathname } from 'next/navigation';
+import React from 'react';
+import ReactCountryFlag from 'react-country-flag';
+
+type Props = {};
+
+const TranslationToggle = (props: Props) => {
+  const pathname = usePathname();
+  const isEnglish = pathname.startsWith('/en');
+  const onClick = () => {
+    setLanguage(pathname);
+  };
+
+  return (
+    <ReactCountryFlag
+      countryCode={isEnglish ? 'DE' : 'GB'}
+      svg
+      style={{
+        width: '1.5em',
+        height: '1.5em',
+        cursor: 'pointer',
+        borderRadius: '50%',
+      }}
```

Comment:
Inline styles aren't very good, it is better to move this object outside the component, otherwise this object will be recreated in the prop every time when component rendering.